corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -653,9 +662,10 @@ def _get_attachment_dicts_from_form(form):
 
 
 @quickcache(['domain'], timeout=24 * 3600)
-def get_case_types_for_domain_es(domain):
+def get_case_types_for_domain_es(domain, use_case_search=False):
+    index_class = CaseSearchES if use_case_search else CaseES
```

Comment:
because locally i was having serious problems with the old case index that I still can't resolve. It also just seems like it should be coming from the same place

Reply:
I don't think this change is necessary. Both of the case indices should return the same results, and this makes it seem like the result will be different. 
I can help you with your local set up so that you will see cases in both indices if that is still an issue.

Reply:
is this a blocking comment if I'm going to be doing work on migrating reports to use only `CaseSearchES` anyway?